### PR TITLE
enhance(webhook): add extra debug logging at various points in workflow

### PR DESCRIPTION
**Pull request by ecrupper**
Closes https://github.com/go-vela/community/issues/657

Trying to add some extra context during the webhook processing without overlapping some of the normal debug logging

### CODE REVIEW

File: api/webhook.go [hunk 1]
```diff
@@ -131,6 +131,10 @@ func PostWebhook(c *gin.Context) {
 
 	h, r, b := webhook.Hook, webhook.Repo, webhook.Build
 
+	logrus.Debugf("Hook generated from SCM: %v", h)
```
[the remark this answers is not in the document]
Comment (plyr4) [reply api/webhook.go:134]: also leaving this here:
https://github.com/go-vela/server/blob/main/api/user.go#L75-L77

you might be able to use fields

File: api/webhook.go [hunk 2]
```diff
@@ -313,6 +320,7 @@ func PostWebhook(c *gin.Context) {
 	}
 
 	// update fields in build object
+	logrus.Debugf("updating build number, parent, and status")
```
[the remark this answers is not in the document]
Comment (ecrupper) [reply api/webhook.go]: Split up!

**Review by plyr4 — DISMISSED** [review]
lgtm ty

**Review by wass3r — APPROVED** [review]

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**